getsolus/packages
# caja: Update to 1.27.2

**Summary**

- 1.27.2 : 
     - Fix gfile sort and symlink warnings w glib2.76 or later
- 1.27.1 : 
     - Update translations 
     - Wayland: initial support for showing the desktop 
     - Ensure all of the desktop is usable in x11 
     - Replace deprecated code from xrandr fix 
     - caja-desktop-window: Fix the xrandr error 
     - caja-icon-container.c: Set a transient parent on the interactive search popup 
     - Initial wayland support 
     - caja-file: fix future informal date bug 
     - caja-file: fix yesterday/today informal date bug 
     - directory: Include hidden files in properties window 
     - Fix handling of invalid x-special/mate-icon-list drops 
     - icon view: Fix positioning overflowing icons on the desktop 
     - Make CajaIconData type less confusing 
     - Remove dead assignment 
     - Remove unused members and structures 
     - Remove unused member 
     -  Enable showing in Budgie Desktop and XFCE 
     - icon view: Refresh icon positions for manual layout on zoom change 
- Added `homepage` key to `package.yml` (Part of #411)

**Test Plan**

- Install package on Budgie VM
- Ensure desktop icon visible from app menu
- Verified the added homepage loads information about the package

**Checklist**
